docs/guides/admin/docs/workflowoperationhandlers/configure-by-dcterm-woh.md

```diff
@@ -15,7 +15,8 @@ Tags and flavors can be used in combination.
 |------------------|-------|-----------|-------------|
 |dccatalog         |"episode" or "series"|the type of catalog in which to search for `dcterm`|EMPTY|
 |dcterm            |"creator"            |the name of the Dublin Core term which to check|EMPTY|
-|match-value       |"Joe Bloggs"         |the Dublin Core term value to check for|EMPTY|
+|externalcatalog   |i.e. "myterms"       |the type of external metadata catalog in which to search for `dcterm`|dublincore|
```

Comment:
From what I understand, this basically defines the main flavor of the catalog to look for in the media package while `dccatalog` is used as sub-flavor. This is quite unusual for a workflow operation. Basically all other operations go with a `source-flavor` option instead and I suggest to use that here as well since that is way less confusing. In addition, the term `dccatalog` doesn't make any sense when not using a Dublin Core catalog.